sonar-ruby-plugin/src/test/java/org/sonarsource/ruby/converter/RubyConverterTest.java

```diff
@@ -109,28 +91,26 @@ public void invalid_escape_sequence() {
     assertThat(converter.parse("\"\\xff\"")).isNotNull();
   }
 
-  @Test
-  public void top_level_tree() {
-    assertTree(converter.parse(("true\nfalse"))).isInstanceOf(TopLevelTree.class);
-    assertTree(converter.parse(("true\r\nfalse"))).isInstanceOf(TopLevelTree.class);
-  }
-
-  @Test
```

Comment:
Looks like this annotation was dropped accidentally, ranges is now not executed as test

Reply:
Nice catch :+1: